python/ycm/vimsupport.py

```diff
@@ -301,6 +301,23 @@ def EscapedFilepath( filepath ):
   return filepath.replace( ' ' , r'\ ' )
 
 
+def TryJumpLocationInOpenedTab( filename, line, column ):
+  bufnr = GetBufferNumberForFilename( filename, False )
+  try:
+    tab, win = next( iter( [ ( tab, win ) for tab in vim.tabpages
+                                          for win in tab.windows
+                                          if tab != vim.current.tabpage
+                                          and win.buffer.number == bufnr ] ) )
+    vim.current.tabpage = tab
+    vim.current.window = win
+    vim.current.window.cursor = ( line, column - 1 )
```

Comment:
Actually now that I checked neither line 322 has the comment, but there is a comment at the top of the function saying that the line and column are 1-based. Where should I add the comment?